os/pm/pm_changestate.c

```diff
@@ -257,4 +257,23 @@ int pm_changestate(int domain_indx, enum pm_state_e newstate)
 	return ret;
 }
 
+/****************************************************************************
+ * Name: pm_querystate
+ *
+ * Description:
+ *   This function returns the current power management state.
+ *
+ * Input Parameters:
+ *   domain - The PM domain to check
+ *
+ * Returned Value:
+ *   The current power management state.
+ *
+ ****************************************************************************/
+
+enum pm_state_e pm_querystate(int domain)
```

Comment:
Some APIs have domain but others have domain_indx as argument. Is it different meaning?

Reply:
No, both are same.
infact domain is more appropriate